jenkins/pipelines/cd/atom-jobs/build-common.groovy

```diff
@@ -1000,7 +999,7 @@ def run_with_arm_go_pod(Closure body) {
 try {
     stage("Build ${PRODUCT}") {
         if (!ifFileCacheExists()) { 
-            if (params.PRODUCT in ["tidb", "enterprise-plugin"] && params.ARCH == "arm64" &&  params.OS == "linux") {
+            if ( params.ARCH == "arm64" &&  params.OS == "linux") {
```

Comment:
This needs to be further modified here, excluding the tiflash arm64 build because tiflash uses a separate image.